playbooks/molecule/resources/monitoring/verify.yml

```diff
@@ -6,7 +6,7 @@
   tasks:
     - name: Check alertmanager exporter container is running
       community.docker.docker_container_info:
-        name: "{{ monitoring_server_alertmanager.container_name }}"
+        name: alertmanager
```

Comment:
the variables are not available to the verify playbook so I've hardcoded the container names